AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -609,7 +609,6 @@ + (instancetype)sharedInstance {
 }
 
 + (void)resetSharedInstance {
-
```

Comment:
This line should be there.

Reply:
I'd bet $5 this is just the formatter being dumb, I doubt it was removed intentionally 😉

Reply:
The formatter managed to bring it back it seems.

┻━┻︵└(՞▃՞ └)